assets/components/portfolio-card/portfolio-card.scss

```diff
@@ -0,0 +1,66 @@
+@import "./../constants";
+
+$FOOTERHEIGHT: 100px;
+$BLOCKHEIGHT: calc(100% - 100px);
```

Comment:
1) You need use $FOOTERHEIGHT in $BLOCKHEIGHT
2) Rename constants with separator '_'